src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
I like it but we have to include the UI guys in this decision.

What do you think? This is the path in the JSON error response. We face the problem that the indices are unreliable because they relate to unordered sets and not the order specified in the JSON.

@Poltergeist @pierlo-upitup